pyproject.toml

```diff
@@ -18,6 +18,13 @@ authors = [
 dependencies = [
   "sunpy[map,net,timeseries,visualization]>=6.0.0",
   "xarray>=2023.12.0",
+  "astropy",
+  "packaging",
```

Comment:
Do we need this one? Does it not come with pip these days?

Reply:
This one refers tp packaging right?